Pod/Classes/SideMenuTransition.swift

```diff
@@ -114,7 +114,7 @@ open class SideMenuTransition: UIPercentDrivenInteractiveTransition, UIViewContr
             view.transform = transform
             if velocity >= 100 || velocity >= -50 && abs(distance) >= 0.5 {
                 // bug workaround: animation briefly resets after call to finishInteractiveTransition() but before animateTransition completion is called.
-                if ProcessInfo().operatingSystemVersion.majorVersion == 8 && percentComplete > 1 - CGFloat(FLT_EPSILON) {
+                if ProcessInfo().operatingSystemVersion.majorVersion == 8 && percentComplete > 1 - CGFloat.ulpOfOne {
```

Comment:
This is only available in Swift 3.1 or higher. Is this PR to update to Xcode 8.3 AND Swift 3.1?